src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
OK, here's a test that fails in IE8: https://jsfiddle.net/74eutos8/embedded/result/